packages/runtime/test-runtime-utils/src/test/insecureUrlResolverTest.spec.ts

```diff
@@ -32,22 +33,45 @@ describe("Insecure Url Resolver Test", () => {
         assert(!!request.headers?.[CreateNewHeader.createNew],
             "Request should contain create new header");
         const url = `${hostUrl}?fileName=${fileName}`;
-        assert.equal(request.url, url, "Request url should match");
+        assert.strictEqual(request.url, url, "Request url should match");
     });
 
     it("Resolved CreateNew Request", async () => {
         const resolvedUrl = await resolver.resolve(request) as IFluidResolvedUrl;
         const documentUrl = `fluid://${new URL(ordererUrl).host}/${tenantId}/${fileName}`;
-        assert.equal(resolvedUrl.endpoints.ordererUrl, ordererUrl, "Orderer url should match");
-        assert.equal(resolvedUrl.url, documentUrl, "Document url should match");
+        assert.strictEqual(resolvedUrl.endpoints.ordererUrl, ordererUrl, "Orderer url should match");
+        assert.strictEqual(resolvedUrl.url, documentUrl, "Document url should match");
     });
 
     it("Test RequestUrl for a data store", async () => {
         const resolvedUrl = await resolver.resolve(request);
         const dataStoreId = "dataStore";
-        const response = await resolver.getAbsoluteUrl(resolvedUrl, dataStoreId);
+        const absoluteUrl = await resolver.getAbsoluteUrl(resolvedUrl, dataStoreId);
 
         const compUrl = `${hostUrl}/${tenantId}/${fileName}/${dataStoreId}`;
-        assert.equal(response, compUrl, "Url should match");
+        assert.strictEqual(absoluteUrl, compUrl, "Url should match");
+    });
+
+    it("Test RequestUrl for longer url", async () => {
```

Comment:
>it("Test RequestUrl for longer url", async () => { [](start = 4, length = 50)

This should say something like "test RequestUrl for url with data object Ids" or something. longer url can just be a url with longer documentId.